test/node/ol/Feature.test.js

```diff
@@ -379,7 +391,8 @@ describe('ol/Feature.js', function () {
       const coordinates = geometryClone.getFlatCoordinates();
       expect(coordinates[0]).to.be(1);
       expect(coordinates[1]).to.be(2);
-      expect(clone.getStyle()).to.be(style);
+      expect(clone.getStyle()).to.be.a(Style);
+      expect(clone.getStyle()).not.to.be(style);
```

Comment:
Previously, cloned features shared a style.  I think this was a bug.

Reply:
That was raised in https://github.com/openlayers/openlayers/issues/7679#issuecomment-355902011  `styles are meant to be shared between features` is valid, so maybe it should be an option as suggested in the reply?